docs/source/tracking/backend-stores.rst

```diff
@@ -0,0 +1,105 @@
+.. _backend-stores:
+
+==============
+Backend Stores
+==============
+
+The backend store is a core component in `MLflow Tracking <../index.html>`_ where MLflow stores metadata for 
+:ref:`Runs <runs>` and experiments such as:
+
+* Run ID
+* Start & end time
+* Parameters
+* Metrics
+* Code version (only if you launch runs from an :ref:`MLflow Project <projects>`).
+* Source file name (only if you launch runs from an :ref:`MLflow Project <projects>`).
+
+Note that large model artifacts such as model weight files are stored in `artifact store <artifacts-stores.html>`_,  the other core component in MLflow Tracking.
```

Comment:
```suggestion
Note that large model artifacts such as model weight files are stored in `artifact store <artifacts-stores.html>`_,  the other core components are stored in your backend store (MLflow Tracking).
```

Reply:
oops I meant sth like "artifact store, which is the other component in MLflow tracking", will rephase a bit